### DISCUSSION TITLE
Conseil "n'oubliez pas de nourrir vos pet" à changer pour ceux qui ont le badge 💞

### DISCUSSION THREAD

**Discussion by Gabrielre14** [Crownicles/Crownicles#872] [Nouvelles idées]
Bonjour à tous
J'ai une petite idée pour améliorer un conseil en particulier, celui disant "N'oubliez pas de nourrir vos familiers". J'ai reçu cette idée lorsque Hitori a vu ce conseil juste après avoir reçu le badge "Personne ayant dressé un familier 5 étoiles".
Je pensais que l'on pourrait remplacer ce conseil par "Bravo d'avoir dressé un familier à 5 étoiles" pour tous les joueurs ayant ce badge. Ne connaissant pas le langage code pour DraftBot, je ne peux décrire un programme à faire, mais cette option peut être facilement réalisé par quelqu'un familiarisé avec la programmation de bots.
Content d'avoir peut-être pu être utile pour le bot 🙂

**Comment by BastLast** [comment]
le ratio relou à dev / effet pour les joueurs n'est pas excellent mais si un contributeur trouve la motiv de le faire why not
